### PR TITLE
Add .jsc files to gitignore

### PR DESCRIPTION
**Pull request by MiladNazeri**
.jsc files are being created and unnecessarily being checked into the repo.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/5812/app_12052_PR.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/5812/HighFidelity-Beta-PR12052-50f788d741b5bbdd521ac306b7fc941fb84abe1f.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/5812/HighFidelity-Beta-PR12052-50f788d741b5bbdd521ac306b7fc941fb84abe1f.dmg

File: .gitignore [hunk 1]
```diff
@@ -67,6 +67,7 @@ gvr-interface/libs/*
 TAGS
 *.sw[po]
 *.qmlc
```
Comment (zfox23) [inline .gitignore]: Why is `*.qmlc` in here twice? This PR would be a good opportunity to clean up our `.gitignore` file by removing the duplicate (if that's possible).

**Comment by ctrlaltdavid** [comment]
interface/resources/qml/js/Utils.jsc that's in the current master (as of build 7612) should be deleted.

**Comment by zfox23** [comment]
Please delete the `Utils.jsc` file as per @ctrlaltdavid's comment, then I'll approve this!

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/5915/app_12052_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/5915/HighFidelity-Beta-PR12052-0b70eba04faf66430fae9ca54b2a54e7389780d3.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/5915/HighFidelity-Beta-PR12052-0b70eba04faf66430fae9ca54b2a54e7389780d3.exe

**Pull request closed without merging**